### PR TITLE
Pending organization pages + several fixes

### LINKED ISSUE #2270: Implement new edit organization user page
screen shot 2015-02-17 at 10 12 15

### LINKED ISSUE #2271: Implement create new organization user page
screen shot 2015-02-17 at 10 20 08

### LINKED ISSUE #2272: Implement new organization profile page
screen shot 2015-02-17 at 10 19 33

### LINKED ISSUE #2273: Implement new organization index page (aka manage seats)
screen shot 2015-03-04 at 11 42 54

### LINKED ISSUE #2446: Explore and Learn tabs in the New dashboard
Hey,

I have been exploring the new dashboard and was wondering if it may be worth it to have the Explore and Learn buttons to open in a new tab so the dashboard is still opened if you are just checking something or looking for some help.

cc @xavijam @saleiva

### LINKED ISSUE #2566: Fix first element margin on the datasets view
screen shot 2015-03-02 at 17 06 09

### LINKED ISSUE #2621: Change section order in the create map modal
After some tests with real users the correct order should be
Connect Dataset | Datasets | Liked Datasets | Data library 

For org users should be
Connect Dataset | Datasets | Shared with you | Liked Datasets | Data library

The only thing that should change is that the selected item when having datasets already imported should be Datasets, and no Connect dataset like when you don't have any data.

Sorry!

### LINKED ISSUE #2648: new dashboard: import won't work with files with extensions in capital letters
steps to reproduce:
- import `GRAN_TRAIL_WEB.GPX`

expected behaviour:
- file imports fine and user is redirected

actual behaviour:
- invalid import: Unfortunately this file extension is not allowed

### PR DESCRIPTION
**Pull request by xavijam**
- [x] Settings
- [x] Manage seats 
- [x] Create/edit organization user 

screen shot 2015-03-04 at 16 49 32
screen shot 2015-03-04 at 16 49 44
screen shot 2015-03-05 at 17 45 07

Fixes #2273
Fixes #2272
Fixes #2271
Fixes #2270
Fixes #2446
Fixes #2621
Fixes #2566
Fixes #2648

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2078/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2081/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2083/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2084/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2096/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2104/))

**Comment by xavijam** [comment]
cc @Kartones

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2105/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2107/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2108/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2110/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2112/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2113/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2114/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2115/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2116/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2117/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2118/))

**Comment by xavijam** [comment]
@juanignaciosl, checking new organization pages, we have seen that any new user starts with 74 maps.

screen shot 2015-03-06 at 10 53 55

Seems like we are "counting" data library maps. Any advice?
We are currently using:

```@user.visualization_count```

**Comment by juanignaciosl** [comment]
I think that's a wrong method for that, because it counts all visualization without taking care of the type. I think you only want `derived`, of course not `remote` but not `table` either (I think). I think this is the line you want: `u.visualization_count({type: CartoDB::Visualization::Member::TYPE_DERIVED})`, but I don't know for sure if you want to `exclude_shared` or `exclude_raster`. You should wrap it in a `u.all_visualization_count` method. Open `user.rb` and take a look at [`public_visualization_count` method](https://github.com/CartoDB/cartodb/blob/master/app/models/user.rb#L1282). You probably want to duplicate that code for your new `all_visualization_count` and remove `privacy` filter.

File: app/controllers/admin/organization_users_controller.rb [hunk 1]
```diff
@@ -24,10 +24,26 @@ def initialize_google_plus_config
...
+    respond_to do |format|
+      format.html { render view, layout: layout }
+    end
   end
 
   def edit
     set_flash_flags
+
+    new_dashboard = current_user.has_feature_flag?('new_dashboard')
+    view =  new_dashboard ? 'new_edit' : 'edit'
```
Comment (Kartones) [inline app/controllers/admin/organization_users_controller.rb]: oh, no! so we'll have a "new_new_edit" on the next redesign? :trollface: 

When everyone has the new dashboard would be nice to rename it to "edit" or something without a temporal connotation.

Comment (xavijam) [inline app/controllers/admin/organization_users_controller.rb]: We will need to make a deep clean, yep

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2120/))

File: app/controllers/admin/organization_users_controller.rb (hunk 1 above)
Comment (juanignaciosl) [inline app/controllers/admin/organization_users_controller.rb]: In Valladolid we have a football stadium called "New Stadium José Zorrilla" back from 1982 :-)

Comment (xavijam) [inline app/controllers/admin/organization_users_controller.rb]: But new_* is always nice, isn't it?

File: app/views/admin/organization_users/edit.html.erb [hunk 2]
```diff
@@ -40,6 +40,7 @@
 
 <article>
   <div class="inner">
+    More info!
```
Comment (xavijam) [inline app/views/admin/organization_users/edit.html.erb]: Removing this thing :-1:

File: app/views/admin/organization_users/new_edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            <i class="iconFont iconFont-Document OrganizationUser-statIcon"></i>
+            <div class="OrganizationUser-statInfo">
+              <em class="OrganizationUser-statTitle"><%= number_with_delimiter(@user.tables.count) %></em>
+              <label class="OrganizationUser-statLabel"><%= "Dataset".pluralize(@user.tables.count) %></label>
+            </div>
+          </li>
+          <li class="OrganizationUser-stat">
+            <i class="iconFont iconFont-Eye OrganizationUser-statIcon"></i>
+            <div class="OrganizationUser-statInfo">
+              <% mapCount = @user.visualization_count({ type: CartoDB::Visualization::Member::TYPE_DERIVED, exclude_shared: true, exclude_raster: true }) %>
```
Comment (xavijam) [inline app/views/admin/organization_users/new_edit.html.erb]: @juanignaciosl, sounds ok?

Comment (juanignaciosl) [inline app/views/admin/organization_users/new_edit.html.erb]: I think so. If numbers match, please hide that inside a new User method, that doesn't belongs to the UI at all.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2121/))

File: app/views/admin/organization_users/new_edit.html.erb (hunk 3 above)
Comment (Kartones) [inline app/views/admin/organization_users/new_edit.html.erb]: yep, something like "owned_visualizations_count". 

BTW, why we exlude raster there? I'd remove that filter, in the public dashboard we don't show raster tables but in that count they should be...

Comment (xavijam) [inline app/views/admin/organization_users/new_edit.html.erb]:
```ruby
# Get user owned visualizations
  def owned_visualizations_count
    visualization_count({
      type: CartoDB::Visualization::Member::TYPE_DERIVED,
      exclude_shared: true,
      exclude_raster: true
    })
  end
```

**Comment by Kartones** [comment]
haven't digged into FE code or CSS but apart from the comments looks :+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2123/))

**Pull request merged**